### PR TITLE
Automated release of 30.1.0a1 upstream release 30.0 1716337924

### PR DESCRIPTION
**Pull request by yliaog**
#### What type of PR is this?
/kind bug

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,11 +158,12 @@ between client-python versions.
 | 26.0 Alpha/Beta | Kubernetes main repo, 1.26 branch    | ✗                             |
 | 26.0            | Kubernetes main repo, 1.26 branch    | ✗                             |
 | 27.0 Alpha/Beta | Kubernetes main repo, 1.27 branch    | ✗                             |
-| 27.0            | Kubernetes main repo, 1.27 branch    | ✓                             |
+| 27.0            | Kubernetes main repo, 1.27 branch    | ✗                             |
```
Comment (roycaihw) [inline README.md:161]: Nit: According to the [doc](https://github.com/kubernetes-client/python#:~:text=support%20three%20GA%20major%20releases) we will drop 27.0 support when 30.0 GA comes out.

Comment (yliaog) [reply README.md]: Fixed

**Comment by roycaihw** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**